src/components/navbar/buttons/hamburger.scss

```diff
@@ -1,25 +1,78 @@
 .hamburger__links {
     overflow: hidden;
-    height: 12rem;
 
     &__ul {
       margin: 0;
       padding: 0;
       box-sizing: border-box;
+      background-color: white;
+      position: absolute;
+      top: 60px;
+      z-index: 10;
+      width: 100%;
+
+      &--menu {
+        height: 33rem;
```

Comment:
Why rems?

Reply:
It is 510px. Just not to have that much big number, I used rem. I will change